frontend/model/contracts/group.js

```diff
@@ -1339,7 +1339,10 @@ sbp('chelonia/defineContract', {
         chatRoomID: stringMax(MAX_HASH_LEN, 'chatRoomID'),
         description: stringMax(CHATROOM_DESCRIPTION_LIMITS_IN_CHARS, 'description')
       })),
-      process ({ data }, { state }) {
+      process ({ data, direction }, { state }) {
+        if (direction === 'incoming') {
+          throw new Error('chatroom description error')
+        }
```

Comment:
This should be removed.

Reply:
Yeah, that certainly shouldn't be there. Good catch.